src/components/AlertSnackbar/AlertSnackbar.tsx

```diff
@@ -1,7 +1,7 @@
 import classes from "./AlertSnackbar.module.scss";
 
 interface IAlertSnackbar {
-  message: string;
+  message: string | undefined
```

Comment:
this is perfectly ok, but if you don't know, you can use 
```suggestion
  message?: string
  ```